_rules/auto-play-audio-does-not-exceed-3-seconds-aaa1bf.md

```diff
@@ -62,15 +62,15 @@ There are no major accessibility support issues known for this rule.
 
 #### Passed Example 1
 
-The `<audio>` element does not play automatically for more than 3 seconds.
+This `audio` element does not play automatically for more than 3 seconds.
 
 ```html
 <audio src="/test-assets/moon-audio/moon-speech.mp3#t=25" autoplay></audio>
 ```
 
 #### Passed Example 2
 
-The `<video>` element audio output does not last longer than 3 seconds.
+This `video` element's `audio` output does not last longer than 3 seconds.
```

Comment:
The audio in this sentence does not refer to an element or attribute so it should not be between backticks.

```suggestion
This `video` element's audio output does not last longer than 3 seconds.
```